### PR TITLE
initial version in English

### PR DESCRIPTION
**Pull request by avelino**
playbook English translation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,34 +259,34 @@ Escolher uma das opções a seguir, definidas no arquivo *internal/errors/errors
...
-	session := r.pool.Session(nil)
-	defer session.Close()
-	coll := session.DB(r.db).C("address")
-	err := coll.Find(bson.M{"_id": id}).One(&result)
-
-	if err != nil {
-		return nil, &errors.Error{Op: "address.MongoRepository.Find", Err: err, Code: errors.ENOTFOUND}
-	}
-	return &result, nil
+ result := entity.Address{}
```
Comment (mfbmina) [inline README.md]: @avelino are these changes expected? I believe that you wanna remove empty spaces but I believe it should not happen here.

Comment (avelino) [reply README.md]: my editor switched automatically, sorry

File: README_EN.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
+# Guia de padrões e boas práticas em Go
```
Comment (eminetto) [inline README_EN.md]:
```suggestion
# The Trybe Go Style Guide.
```

Comment (avelino) [reply README_EN.md]: sorry

**Review by eminetto — APPROVED** [review]

**Comment by avelino** [comment]
Thank you for accepting my contribution

**Pull request merged**